src/components/About.js

```diff
@@ -72,7 +80,7 @@ const About = (props) => {
   return (
     <Paneset>
```

Comment:
The About page should probably be columns that stack vertically at smaller sizes instead of the responsive behavior of `Paneset`s, but we can tackle that in a future pull request.